docs/rfc/rfc-1.adoc

```diff
@@ -1010,8 +1015,10 @@ Alphabetized list of Governable Parameters with additional notes.
 - `wallet_creation_period`: How frequently we attempt to create new wallets.
 - `wallet_max_age`: The oldest we allow a wallet to become before we transfer
   the funds to a randomly selected wallet.
-- `wallet_min_btc`: The smallest amount of BTC a wallet can hold before we
-  attempt to close the wallet and transfer the funds to a randomly selected
+- `wallet_min_closure_btc`: The smallest amount of BTC a wallet can hold before
+  we attempt to close the wallet and transfer the funds to a randomly selected
   wallet.
+- `wallet_min_creation_btc`: The minimum amount of BTC an active wallet needs
+  to have before we allow for the creation of a new active wallet.
```

Comment:
We should add `wallet_dust_leftover` to this list.

Reply:
[7107234](https://github.com/keep-network/tbtc-v2/pull/183/commits/71072349b551700ab4b7a17a28e14ea5dd936bab)